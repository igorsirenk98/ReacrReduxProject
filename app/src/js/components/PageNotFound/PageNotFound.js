import React from 'react';
import { Link } from 'react-router-dom';

import CssBaseline from '@material-ui/core/CssBaseline';
import Container from '@material-ui/core/Container';
import Typography from '@material-ui/core/Typography';
import Button from '@material-ui/core/Button';

const PageNotFound = () => (
    <>
        <CssBaseline />
        <Container className="container center" fixed>
            <Typography variant="h3" noWrap>Oops, page not found</Typography>
            <iframe src="https://giphy.com/embed/R8ygnAs1Qsdr2" frameBorder="0" className="gif" allowFullScreen></iframe>
            <Button>
                <Link
                    to={{ pathname: '/' }}
                    className="link link-blue"
                >
                    Back to main page
                </Link>
            </Button>
        </Container>
    </>
);

export default PageNotFound;