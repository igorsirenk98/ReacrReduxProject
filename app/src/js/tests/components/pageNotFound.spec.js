import React from 'react'
import { shallow } from 'enzyme'
import PageNotFound from '../../components/PageNotFound/PageNotFound';

describe('PageNotFound component', () => {
    const pageNotFound = shallow(<PageNotFound />);

    it('renders correctly', () => {
        expect(pageNotFound.debug()).toMatchSnapshot()
    })
})
