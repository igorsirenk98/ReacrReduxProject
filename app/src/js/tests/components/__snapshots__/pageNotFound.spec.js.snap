// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound component renders correctly 1`] = `
"<Fragment>
  <CssBaseline />
  <WithStyles(ForwardRef(Container)) className=\\"container center\\" fixed={true}>
    <WithStyles(ForwardRef(Typography)) variant=\\"h3\\" noWrap={true}>
      Oops, page not found
    </WithStyles(ForwardRef(Typography))>
    <iframe src=\\"https://giphy.com/embed/R8ygnAs1Qsdr2\\" frameBorder=\\"0\\" className=\\"gif\\" allowFullScreen={true} />
    <WithStyles(ForwardRef(Button))>
      <Link to={{...}} className=\\"link link-blue\\">
        Back to main page
      </Link>
    </WithStyles(ForwardRef(Button))>
  </WithStyles(ForwardRef(Container))>
</Fragment>"
`;
